File: 프로그래머스/1/12916. 문자열 내 p와 y의 개수/문자열 내 p와 y의 개수.java
import java.util.*;

class Solution {
    boolean solution(String s) {
        s = s.toLowerCase();
        int count = 0;
        
        for (int i = 0; i < s.length(); i++) {
            if (s.charAt(i) == 'p') {
                count++;
            } else if (s.charAt(i) == 'y') {
                count--;
            }
        }
        if (count == 0) return true;
        else return false;
    }
}

File: 프로그래머스/1/12916. 문자열 내 p와 y의 개수/문자열 내 p와 y의 개수.js
function solution(s){
    let arr = s.toLowerCase().split('');
    let count = 0;
    arr.forEach(e => {
        if (e === 'p') {
            count++;
        } else if (e === 'y'){
            count--;
        }
    });

    return count === 0;
}